src/app/widgets/inlineinput/inlineinput.component.ts

```diff
@@ -83,12 +83,16 @@ export class InlineInputComponent implements OnInit {
     }
   }
 
+  isFloat(x) {
+    return isFinite(x) && x == Math.fround(x);
```

Comment:
This is failing. Try `1.2`.

I would suggest to use `Number.isFinite()` instead of `isFinite()` because `isFinite(null)` is `true` where as `Number.isFinite(null)` is `false`.

Reply:
My Bad, I've updated the function and added 5 new test cases. Can you check if this suffice or if the allowed range needs to be changes?